recipes/ibm_db/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "ibm_db" %}
```

Comment:
Looks like the official name on pypi is with a `-`. Should we also keep this?

Reply:
That's how the actual file is named.  An earlier version of the meta.yaml had that name you suggested, but it didn't propagate correctly.  When importing packages, underscores are used.  Please advise.

Reply:
As the documentation of `ibm_db` also is always using an `_` and the sdist is using, we can keep this here.

Note that the `conda` package name and the Python module name can be totally different.